### ISSUE TITLE
最新代码test_loss=nan?

### ISSUE THREAD

**Issue by ITKing80** [YunYang1994/tensorflow-yolov3#149]
生成的文件名是yolov3_test_loss=nan.ckpt-3.index
这是怎么回事？

**Comment by shoutOutYangJie** [comment]
请问你解决了吗？作者估计更新之后有了新bug，我也是这个问题。而且loss在13左右下不去了

**Comment by potter2010** [comment]
> 请问你解决了吗？作者估计更新之后有了新bug，我也是这个问题。而且train loss在13左右下不去了，test loss 为Nan

1. 多训练几个EPOCH试试，我训练的前几轮也出现过NAN，后面正常；2. 如果不行，根据作者在其他issue上说，可以适当提高__C.TRAIN.WARMUP_EPOCHS；3. 可以试着把__C.TEST.BATCH_SIZE设为1训练看看
